pkg/kobject/kobject.go

```diff
@@ -83,6 +83,7 @@ type ServiceConfig struct {
 	Stdin         bool                `compose:"stdin_open" bundle:""`
 	Tty           bool                `compose:"tty" bundle:""`
 	MemLimit      yaml.MemStringorInt `compose:"mem_limit" bundle:""`
+	TmpFs         []string            `compose:"tmpfs" bundle:""`
```

Comment:
Nevermind! I confirmed that both work regardless of string slice or normal string.